ItemGenerator/WordGenerator.cs

```diff
@@ -23,12 +22,10 @@ public string Generate(WordSequenceItem item, int seed)
                 if (item.Words == null || !item.Words.Override)
                     wordList.AddRange(_globalConfiguration.DefaultWords);
 
-                targetWord = wordList[randomSeed.Next(wordList.Count)];
+                targetWord = wordList[cryptoRandom(0, wordList.Count-1)];

```

Comment:
This confuses me, does it compile? (I can not try it myself yet)
Either missing the `.GetRandomInRange` or delegate definition, altough if a delegate is defined, might as well use it throughout...